docs/4.3/aws-oss-guide.md

```diff
@@ -93,13 +91,27 @@ Setting Stream to `NEW IMAGE`
 For maintainability and ease of use, we recommend following our [Terraform example](https://github.com/gravitational/teleport/blob/master/examples/aws/terraform/ha-autoscale-cluster/dynamo.tf)
 but below are high level definitions for the tables required to run Teleport.
 
-Cluster State:
+Table 1 - Cluster State:
```

Comment:
Maybe add a reference here that this is `DYNAMODB-RESOURCE-A` and table 2 is `DYNAMODB-RESOURCE-B`? It's kinda confusing, I'm not 100% sure why we renamed them.

Reply:
I've updated the title to both focus on the below Table and the fact this is a DynamoDB Table